webviz_subsurface/plugins/_pvt_plot/_plugin_ids.py

```diff
@@ -0,0 +1,18 @@
+class PluginIds:
+    # pylint: disable=too-few-public-methods
```

Comment:
It's enough to add this comment once to the top of the file. You can remove all the other comments.